pkg/storage/azure/azure_test.go

```diff
@@ -722,6 +722,7 @@ func Test_assureContainer(t *testing.T) {
 			drv := NewDriver(context.Background(), storageConfig, listers)
 			drv.authorizer = autorest.NullAuthorizer{}
 			drv.sender = sender
+			primaryKey = cachedKey{}
```

Comment:
is it in order to clear cache after another test?

Reply:
It does not change much, with this in place we can avoid having different mocked http responses when the key is cached. It gives all tests on that test table the same environment.